test/CMakeLists.txt

```diff
@@ -1,20 +1,22 @@
-# Tests
-ament_add_gtest(controller_test
-  controller_test.cpp
-)
-ament_target_dependencies(controller_test
-  ${dependencies_pkgs}
-)
-target_link_libraries(controller_test
-  mppic
+set(TEST_NAMES
+  optimizer_test
+  controller_test
 )
 
-ament_add_gtest(optimizer_test
-  optimizer_test.cpp
+
+foreach(name IN LISTS TEST_NAMES)
+ament_add_gtest(${name}
```

Comment:
Missing some indentation I suspect the linter will have an issue with

Reply:
Actually linter has no issue. Anyway fixed